.github/workflows/test.yml

```diff
@@ -31,11 +32,23 @@ jobs:
         run: |
           ./gradlew distributeSql
           ./gradlew examples:banananation:build
+        env:
```

Comment:
Are these needed for gradle?

Reply:
Should be able to safely remove these :)